ros_buildfarm/config/plot_config.py

```diff
@@ -46,6 +46,10 @@ def __init__(self, name, data):
         self.y_axis_minimum = data.get('y_axis_minimum', None)
         self.y_axis_maximum = data.get('y_axis_maximum', None)
 
+        self.num_build = 10
+        if 'num_build' in data:
+            self.num_build = data['num_build']
```

Comment:
It might be good to check the type of the value to ensure it will work as a plugin config option.

Reply:
I included an `assert` maybe is it better to cast the value to `int`?

Reply:
The documentation states that you can pass an empty string which means to keep all builds. That is still not reflected here.